tools/Custom/HttpMessageLogFormatter.cs

```diff
@@ -44,7 +44,7 @@ public static string GetHttpResponseLog(HttpResponseMessage response)
             string body = string.Empty;
             try
             {
-                body = (response.Content == null) ? string.Empty : FormatString(response.Content.ReadAsStringAsync().Result);
+                body = (response.Content == null) ? string.Empty : FormatString(response.Content.ReadAsStringAsync().GetAwaiter().GetResult());
             }
             catch { }
```

Comment:
What happens when there is an exception, such as a network error?

Reply:
Transport exceptions will be handled by `OnCmdletException` - https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/bugfixes/SerializationDepth/tools/Custom/Module.cs#L68-L70.
![image](https://user-images.githubusercontent.com/7061532/146057811-ddd445d0-dcab-45ac-b1c3-01dfb6135ec8.png)

Here, we are just logging an empty body if we can't read the body content for whatever reason. The code should still flow, but the logs will have an empty body.

Reply:
I just noticed that AutoREST is not catching HTTP request timeouts. My last commit fixes this by adding a `catch(System.Exception ex)` in `ProcessRecordAsync` method. Only `PipelineStoppedException`, `AggregateException`, `UndeclaredResponseException`, and `RestException` were caught.